fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -865,8 +865,6 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VariableMgr.VarAttr(name = CHUNK_SIZE, flag = VariableMgr.INVISIBLE)
     private int chunkSize = 4096;
```

Comment:
what abot `batch_size`, clean it?

Reply:
how about adding a check in FE? just return error if chunk_size is not in range of (0,65535]

Reply:
Ok, seems better.

Reply:
> what abot `batch_size`, clean it?

batch_size is marked as deprecated.